infra/main.bicep

```diff
@@ -339,6 +340,7 @@ output AZURE_RESOURCE_GROUP string = rg.name
 output AZURE_TENANT_ID string = tenant().tenantId
 output AZURE_AIPROJECT_CONNECTION_STRING string = projectConnectionString
 output AZURE_AI_AGENT_DEPLOYMENT_NAME string = agentDeploymentName
+output AZURE_AI_AGENT_NAME string = agentName
```

Comment:
Let us propagate the variable to the `api.bicep`, in this case we can avoid running load_dotenv() in the gunicorn script.

Reply:
Let's do this after this PR has been merged